packages/tooltips/examples/default.md

```diff
@@ -0,0 +1,100 @@
+The `Tooltip` component acts as a high-abstraction API over the
+[useTooltip()](https://www.npmjs.com/package/@zendeskgarden/container-tooltip)
+hook.
+
+All props passed to the root element are proxied onto the tooltip element.
```

Comment:
Can we clarify with "All ~props~ events and attributes ~passed~ applied to the root element ..."